src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyDefinition.cs

```diff
@@ -141,17 +141,37 @@ private JToken GetResource()
             var policyDefinitionObject = new PolicyDefinition
             {
                 Name = this.Name,
-                Properties = new PolicyDefinitionProperties
-                {
-                    Description = this.Description ?? null,
-                    DisplayName = this.DisplayName ?? null,
-                    PolicyRule = this.GetObjectFromParameter(this.Policy, nameof(this.Policy)),
-                    Metadata = this.Metadata == null ? null : this.GetObjectFromParameter(this.Metadata, nameof(this.Metadata)),
-                    Parameters = this.Parameter == null ? null : this.GetObjectFromParameter(this.Parameter, nameof(this.Parameter)),
-                    Mode = string.IsNullOrEmpty(this.Mode) ? PolicyDefinitionMode.All : this.Mode
-                }
+                Properties = new PolicyDefinitionProperties()
             };
 
+            var policyObject = this.GetObjectFromParameter(this.Policy, nameof(this.Policy));
+            if (policyObject["policyRule"] != null)
```

Comment:
Have we had requests for supporting this?  If this starts being supported it should be supported in Set-AzPolicyDefinition as well

Reply:
Working from a work item you opened a while back. I think there were requests.